src/content/advanced/prometheus-remotewrite/index.md

```diff
@@ -0,0 +1,66 @@
+---
+linkTitle: Prometheus RemoteWrite
+title: Prometheus RemoteWrite
+description: A guide explainig what the Giant Swarm Prometheus Remote Write feature is and how to use it.
+weight: 50
+menu:
+  main:
+    parent: advanced
+user_questions:
+  - What is a Prometheus RemoteWrite?
+  - How Prometheus RemoteWrite work?
+  - How do I get access to management cluster metrics?
+aliases:
+  - /advanced/remotewrite/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2022-07-19
+---
+
+In this document you will learn how to use the Prometheus RemoteWrite feature provided out of the box by Giant Swarm.
+
+## Introduction to remote write is Prometheus RemoteWrite
+
+Prometheus allows users to replicate it's data into 3rd party systems like Cortex or even another Prometheus using its [remote APIs](https://prometheus.io/blog/2019/10/10/remote-read-meets-streaming/#remote-apis).
+To enable this behavior, Prometheus needs to be configured using [remote_write config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) to define which endpoints to send data to.
+
+## What is the Giant Swarm Prometheus RemoteWrite feature
+
+The Prometheus `RemoteWrite` feature that allows you to configure remote write targets in the Giant Swarm managed prometheus set up on your management cluster.
+That way, you can replicate and send metrics to your own 3rd party system and use the data as you see fit for the component running on the management clusters such as Flux
```

Comment:
I would remove the Flux example here, it does not give a concrete enough example, and its confusing to talk about management cluster concept here IMO.

```suggestion
That way, you can replicate and send metrics to your own 3rd party system and use the data as you see fit.
```

Reply:
Well the feature is only available on the management clusters and was made for flux originally :D but sure